### PR TITLE
Added new screen that is showing users presence over their entire wor…

### PR DESCRIPTION
**Pull request by RadekLejba**
…k time

### CODE REVIEW

**Comment by RadekLejba** [comment]
https://kindergarten.stxnext.pl/browse/WDP-77

File: src/presence_analyzer/static/js/monthly_presence.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                        options = {
+                            curveType: 'function',
+                        };
+
+                    $chart_div.show();
+                    $loading.hide();
+                    chart.draw(data, options);
+
+                    $.getJSON("/api/v1/users/"+selected_user, function(result) {
+
```
Comment (dpolcyn) [inline src/presence_analyzer/static/js/monthly_presence.js]: Fix this empty lines.

File: src/presence_analyzer/views.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 from presence_analyzer.main import app
 from presence_analyzer.utils import jsonify, get_data, mean, group_by_weekday
 from presence_analyzer.utils import group_by_start_end, get_xml_data
+from presence_analyzer.utils import group_by_months
```
Comment (dpolcyn) [inline src/presence_analyzer/views.py]: you have presence_analyzer.utils 1 line below

File: src/presence_analyzer/utils.py [hunk 3]
```diff
@@ -153,6 +153,40 @@ def group_by_start_end(items):
...
+                str(date.year),
+                str(date.month)
+            )
+        else:
+            key = '{}.{}'.format(
+                str(date.year),
+                str(date.month)
+            )
+
+        if key not in result:
```
Comment (dpolcyn) [inline src/presence_analyzer/utils.py]: if key in result:
    result[key] += time
....

**Pull request merged**